compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -46,12 +46,35 @@ int32_t get_broadcastTo_index_among_inputs_of(luci::CircleCustom *cop)
         bc_idx = static_cast<int32_t>(idx);
       }
     }
+    else
+    {
+      auto broadcastTo = dynamic_cast<luci::CircleBroadcastTo *>(cop->inputs(idx));
+      if (broadcastTo)
+        return idx;
```

Comment:
how does `This function assumes there is only one BroadcastTo node among its inputs.` condition check here?

Reply:
@Seunghui98 Hmm.. I think there are two things to consider.

1. Using opcode

Dynamically casting two or more class make the check complicated. If you rather check their opcodes and use them with `must_cast`, the logic looks simpler.
```cxx
// ..
auto input = must_cast<const luci::CircleNode *>(cop->inputs(idx));
switch (input->opcode())
{
  case luci::CircleOpcode::BROADCAST_TO:
  {
    auto broadcastTo = loco::must_cast<luci::CircleBroadcastTo *>(input->input());
    // ..
  }
  case luci::CircleOpcode::CUSTOM
  {
    auto broadcastTo = loco::must_cast<luci::CircleCustom *>(input->input());
    // ..
  }
}
```

2. Introduce a function that checks the assumption

As two or more classes are being considered, assumption check might as well become a dedicated function. Otherwise, the logic would be complicated because current function does two things.

```cxx
int32_t get_broadcastTo_index_among_inputs_of(luci::CircleCustom *cop)
{
  if (num_of_broadcast_to(cop) != 1) return -1;
  // or
  if (not has_single_broadcast_to(cop)) return -1;
  
  // ..
}
```

Reply:
@seanshpark 

>how does This function assumes there is only one BroadcastTo node among its inputs. condition checked here?

no, there isn't condition checked. 😅
so, As in the comment given by @mhs4670go , I will introduce a function that pass can check condition of the inputs. :)
Thank you for letting me know !!

Reply:
@mhs4670go 

Thank you for your comments !!
I will reflect these in this PR and upload again. :)